atoma-proxy/src/server/handlers/chat_completions.rs

```diff
@@ -178,6 +181,8 @@ pub async fn chat_completions_create(
             Err(e) => {
                 let model_label: String = metadata.model_name.clone();
                 TOTAL_FAILED_CHAT_REQUESTS.add(1, &[KeyValue::new("model", model_label)]);
```

Comment:
Why not add a `user_id` label to the `TOTAL_FAILED_CHAT_REQUESTS` as opposed to creating a new metric for it?